MSBUILD: error MSB1001: Unknown switch.

**Godot version:**
master - 12482bf

**OS/device including version:**
Windows 10

**Issue description:**
Compiling with mono fails with MSBUILD error: 


```
build_godot_tools(["bin\GodotSharp\Tools\GodotTools.dll"], ["bin\GodotSharp\Api\Release\GodotSharp.dll", "bin\GodotSharp\Api\Release\GodotSharp.pdb", "bin\GodotSharp\Api\Release\GodotSharp.xml", "bin\GodotSharp\Api\Release\GodotSharpEditor.dll", "bin\GodotSharp\Api\Release\GodotSharpEditor.pdb", "bin\GodotSharp\Api\Release\GodotSharpEditor.xml"])
NuGet path: c:/Program Files/Mono\bin\nuget.bat
MSBuild auto-detection: using msbuild version '15.0' from 'c:\Program Files\Mono\lib\mono\msbuild\15.0\bin'.
WARNING: Error reading msbuild project information, ensure that your input solution or project file is valid. NETCore and UAP projects will be skipped, only packages.config files will be restored.
All packages listed in packages.config are already installed.
MSBuild path: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1001: Unknown switch.
Switch: /Target=BuildAlwaysCopyToDataDir

For switch syntax, type "MSBuild -help"
scons: *** [bin\GodotSharp\Tools\GodotTools.dll] RuntimeError : 'msbuild' exited with error code: 1
Traceback (most recent call last):
  File "C:\dev\godot\modules\mono\build_scripts\solution_builder.py", line 150, in run_command
    subprocess.check_call(args, env=env_override)
  File "C:\Users\<user>\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe', 'C:\\dev\\godot\\modules\\mono\\editor/GodotTools/GodotTools.sln', '/p:Configuration=Release', '/Target=BuildAlwaysCopyToDataDir']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\<user>\appdata\local\programs\python\python36\lib\site-packages\scons\SCons\Action.py", line 1209, in execute
    result = self.execfunction(target=target, source=rsources, env=env)
  File "C:\dev\godot\modules\mono\build_scripts\godot_tools_build.py", line 21, in build_godot_tools
    build_solution(env, solution_path, build_config, extra_build_args)
  File "C:\dev\godot\modules\mono\build_scripts\solution_builder.py", line 214, in build_solution
    run_command(msbuild_path, msbuild_args, env_override=msbuild_env, name='msbuild')
  File "C:\dev\godot\modules\mono\build_scripts\solution_builder.py", line 152, in run_command
    raise RuntimeError("'%s' exited with error code: %s" % (name, e.returncode))
RuntimeError: 'msbuild' exited with error code: 1
scons: building terminated because of errors.

```

Possibly related to https://github.com/godotengine/godot/commit/86274b9fc9e63c0eb6112bf4d87d67dd97fb0b86
**Steps to reproduce:**
Windows 10
Mono JIT compiler version 6.6.0 (Visual Studio built mono)


pull latest
```
rm -rf ./bin/*

# Build temporary binary
scons p=windows tools=yes module_mono_enabled=yes mono_glue=no
# Generate glue sources
bin\godot.windows.tools.64.mono --generate-mono-glue modules/mono/glue

rm ./bin/*

### Build binaries normally
# Editor
scons p=windows target=release_debug tools=yes module_mono_enabled=yes
# Export templates
scons p=windows target=release_debug tools=no module_mono_enabled=yes
scons p=windows target=release tools=no module_mono_enabled=yes
```
Also tried `scons -c` before building

--- a/modules/mono/build_scripts/godot_tools_build.py
+++ b/modules/mono/build_scripts/godot_tools_build.py
@@ -14,7 +14,7 @@ def build_godot_tools(source, target, env):
     build_config = 'Debug' if env['target'] == 'debug' else 'Release'
 
     # Custom build target to make sure output is always copied to the data dir.
-    extra_build_args = ['/Target=BuildAlwaysCopyToDataDir']
+    extra_build_args = ['/Target:Build;GodotTools:BuildAlwaysCopyToDataDir']
 
     from . solution_builder import build_solution, nuget_restore
     nuget_restore(env, solution_path)

